JS-test.js

```diff
@@ -0,0 +1,48 @@
+// 1 Programming Basics
+
+let x
+for (x = 10; x < 40; x++) {
+  if (x % 5 == 0 && x % 3 == 0) {
+    console.log(x + ' Jackpot!')
+  } else if (x % 3 == 0) {
+    console.log(x + ' This is divisible by 3')
+  } else if (x % 5 == 0) {
+    console.log(x + ' This is divisible by 5')
+  } else {
+    console.log(x)
+  }
+}
+
+// 2 DOM manipulation
+let body = document.querySelector('body')
+let button = document.createElement('button')
+button.appendChild(document.createTextNode('Button'))
+button.onclick = function () {
+  const h1 = createElement('h1')
+  h1.innerHTML = 'This is an h1 tag'
+  body.appendChild(h1)
+}
+
+// 3. Async API calls
+
+async function test() {
+  const api = `https://catfact.ninja/fact`
+  const response = await fetch(api)
+  const data = await response.json()
+  for (let i = 0; i < 3; i++) {
+    console.log(data.answer)
+  }
+}
+test()
```

Comment:
The data fetching is very well written! I also like how you sued `async/await` instead of simple `fetch`. You should have added paragraph elements to the document though, using the same technique as in exercise 2. Also, you are missing the `if` statement that the exercise asks for. This function should have been run 3 times, and without a for loop in the function.